src/hooks/useTokenAllowance.ts

```diff
@@ -15,3 +18,34 @@ export function useTokenAllowance(token?: Token, owner?: string, spender?: strin
     [token, allowance]
   )
 }
+
+export function useUpdateTokenAllowance(amount?: CurrencyAmount<Token>, spender?: string) {
```

Comment:
filename and function name should be consistent

Reply:
this is a setter/getter file, so I make an exception